srml/council/src/lib.rs

```diff
@@ -14,7 +14,158 @@
 // You should have received a copy of the GNU General Public License
 // along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
 
-//! Council system: Handles the voting in and maintenance of council members.
+//! # Council Module
+//!
+//! The Countil module provides tools to manage the council and proposals.
+//!
+//! - **Seats**
+//! 	- [`seats::Trait`](./seats/trait.Trait.html)
+//! 	- [`Call`](./seats/enum.Call.html)
+//! 	- [`Module`](./seats/struct.Module.html)
+//! - **Motions**
+//! 	- [`motions::Trait`](./motions/trait.Trait.html)
+//! 	- [`Call`](./motions/enum.Call.html)
+//! 	- [`Module`](./motions/struct.Module.html)
+//! - **Voting**
+//! 	- [`voting::Trait`](./voting/trait.Trait.html)
+//! 	- [`Call`](./voting/enum.Call.html)
+//! 	- [`Module`](./voting/struct.Module.html)
+//!
+//! ## Overview
+//!
+//! The Council module provides functionality to handle:
+//!
+//! - The voting in and maintenance of council members.
+//! - Proposing, vetoing, and passing motions.
+//!
+//! The Council's role is to represent passive stakeholders. The Council is an on-chain entity comprised of
+//! a set of account IDs. Its primary tasks are to propose sensible referenda and thwart any uncontroversially
+//! dangerous or malicious referenda.
+//!
+//! ### Terminology
+//!
+//! #### Council Proposals
+//!
+//! - **Council motion:** A mechanism used to enact a proposal.
```

Comment:
Good thing we have a style guide, which says no blank lines between list items :)
https://github.com/paritytech/substrate-developer-hub/blob/master/style-guide.md#markdown-lists

Reply:
> do we want to capitalize the first letter of each main word in each term for consistency with https://docs.substrate.dev/docs/glossary. i.e. `Council Motion` instead of `Council motion`

I don't really have strong feelings either way. Bias is toward no because some of them are quite long and not all proper nouns.